litemapy/schematic.py

```diff
@@ -198,6 +206,14 @@ def length(self):
             return 0
         return self.__zmax - self.__zmin + 1
 
+    @property
+    def preview(self):
+        return self.__preview
+
+    def set_preview(self, preview):
```

Comment:
Why not use the setter decorator ?

You could do something like this:
```python
@preview.setter
def preview(self, value):
    self.__preview = value
```

It would be more logical do to something like that imo since preview is already a property.
This way the semantic for getting and setting the field would be the same.

Reply:
True. The preview was the first thing I added and only learned about setter later on. Will update